tests/check_integration.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+composer run-script integration-tests --timeout=0;
+INTEGRATION=$?
+
+if [[ "$INTEGRATION" == "0" ]]; then
+  echo -e "\e[INTEGRATION TESTS OK"
```

Comment:
```suggestion
  echo -e "\e[92mINTEGRATION TESTS OK\e[0m"
```

Reply:
alright but I need to modify it for test_unit as well